libnopegl/src/gpu_ctx.h

```diff
@@ -43,12 +43,12 @@ struct scissor {
 
 int ngli_viewport_is_valid(const struct viewport *viewport);
 
-#define NGLI_FEATURE_COMPUTE                           (1 << 0)
-#define NGLI_FEATURE_SOFTWARE                          (1 << 4)
-#define NGLI_FEATURE_IMAGE_LOAD_STORE                  (1 << 9)
-#define NGLI_FEATURE_STORAGE_BUFFER                    (1 << 10)
-#define NGLI_FEATURE_DEPTH_STENCIL_RESOLVE             (1 << 11)
-#define NGLI_FEATURE_BUFFER_MAP_PERSISTENT             (1 << 14)
+#define NGLI_FEATURE_SOFTWARE                          (1 << 0)
+#define NGLI_FEATURE_COMPUTE                           (1 << 1)
```

Comment:
what's the new order based on?

Reply:
"yolo". The reason behind it: first software, then proper graphics features.